modules/apps/announcements/announcements-web/src/main/resources/META-INF/resources/announcements/configuration.jsp

```diff
@@ -21,7 +21,7 @@ String tabs1 = ParamUtil.getString(request, "tabs1", "sites");
 
 List<Group> groups = GroupLocalServiceUtil.getUserGroups(user.getUserId(), true);
 List<Organization> organizations = OrganizationLocalServiceUtil.getUserOrganizations(user.getUserId());
-List<Role> roles = RoleLocalServiceUtil.getRoles(PortalUtil.getCompanyId(renderRequest));
```

Comment:
Why do we have to filter the roles this way and not the organizations or the groups, for example??

Does it make sense to restrict the portlet configuration based on the user that is configuring it?? I don't think so.

Reply:
Groups/organizations/user groups are currently already filtered by user. Roles is the only option that isn't filtered by userId currently in the configuration menu and instead displays all role options. Do you believe it'd be better to change groups/organizations/user groups to display all configuration options instead?

Reply:
I've check the history of this logic and it seems that it was introduced in SOS-1586

Saying this, I think that we should not change this behavior in a fix.

Please, remove this commit and fwd it to @sergiogonzalez 

If you have any concern you could open a new PTR or a PSM if you feel it as a perceived bug

Thanks!!